tempesta_fw/http.c

```diff
@@ -1866,10 +1884,11 @@ tfw_http_resp_fwd(TfwHttpReq *req, TfwHttpResp *resp)
 		return;
 	}
 	BUG_ON(list_empty(&req->msg.seq_list));
-	req->resp = (TfwHttpMsg *)resp;
+	resp->flags |= TFW_HTTP_RESP_READY;
+	WARN_ON((TfwHttpReq *)resp->req != req);
```

Comment:
Looks like redundant WARN_ON here (exactly the same assignment is made several lines above). Or maybe `WARN_ON(req->resp != resp);` should be here.